spec/features/practice_editor/sidenav_spec.rb

```diff
@@ -16,7 +16,7 @@
         end
 
         it 'should be there' do
-            expect(page).to have_css('.usa-sidenav__item', count: 12)
+            expect(page).to have_css('.usa-sidenav__item', count: 13)
```

Comment:
the side nav should not have changed. please revert this test when you revert the sidenav file